geometry_msgs/Point[] point_robot
